File: ios/crypto/Wallet/WalletCryptoBoxBuilder.swift
//
//  WalletCryptoBoxBuilder.swift
//  Openid4vpBle
//
//  Created by Shaik mohammed Jaffer on 24/12/22.
//  Copyright © 2022 Facebook. All rights reserved.
//

import Foundation

@available(iOS 13.0, *)
class WalletCryptoBoxBuilder {
    func build() -> WalletCryptoBox {
        return WalletCryptoBoxImpl()
    }
}

File: android/src/main/java/io/mosip/cryptography/WalletCryptoBoxBuilder.java
package io.mosip.cryptography;

import java.security.SecureRandom;

public class WalletCryptoBoxBuilder {
    public static WalletCryptoBox build(SecureRandom secureRandom) {
        return new WalletCryptoBoxImpl(secureRandom);
    }
}
